components/brave_wallet/browser/brave_wallet_p3a.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wallet/browser/brave_wallet_p3a.h"
```

Comment:
Please insert a blank line here to follow the style guide https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

Reply:
I wonder why the linter/formatter didn't correct this.  I noticed when I run clang-format using the "google" rules, the newline is automatically added. When I run clang-format with the "chromium" rules, the newline is not added (it's also not removed).  Will make the change